code/src/App.js

```diff
@@ -1,9 +1,39 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react';
+import { BrowserRouter, Switch, Route } from "react-router-dom";
+
+import MovieList from "./components/MovieList";
+import MovieDetails from "./pages/MovieDetails";
+import { MOVIES_API } from "./API_info";
 
 export const App = () => {
+
+  const [movies, setMovies] = useState([]);
+
+  //fetch top rated movies
+  useEffect(() => {
+    fetch(MOVIES_API)
+      .then(response => response.json())
+      .then(data => setMovies(data.results));
+
+  }, []);
+
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
-  )
-}
+    <BrowserRouter>
+      <Switch>
+        <Route path="/" exact>
+          <section className="app-container">
```

Comment:
Not a bug, but from semantic point of view your `<MovieList />` is not actually z movie list ;)

Since you generate each `<MovieList />` component inside `.map()` method, it should rather be called `<MovieElement />` or just `<Movie />` :

```
{movies.map(movie => (
              <MovieElement
                key={movie.id}
                {...movie}
              />
            ))}
```